pkg/shp/cmd/build/delete.go

```diff
@@ -2,29 +2,37 @@ package build
 
 import (
 	"errors"
+	"fmt"
 
 	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
 	"github.com/spf13/cobra"
 
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
 	"github.com/shipwright-io/cli/pkg/shp/cmd/runner"
 	"github.com/shipwright-io/cli/pkg/shp/params"
+	"github.com/shipwright-io/cli/pkg/shp/resource"
 )
 
 // DeleteCommand contains data provided by user to the delete subcommand
 type DeleteCommand struct {
 	name string
 
-	cmd *cobra.Command
+	cmd        *cobra.Command
+	deleteRuns bool
 }
 
 func deleteCmd() runner.SubCommand {
 	deleteCommand := &DeleteCommand{
 		cmd: &cobra.Command{
-			Use:   "delete [flags] name",
+			Use:   "delete [flags] [name]",
```

Comment:
I wanted it to be in brackets, because unlike `delete` it's a variable, or a parameter, not subcommand.
But, I have just looked through `man` pages conventions, and it seems, that angular brackets (`<>`) might be a better choice here, because unlike square brackets (`[]`), which represent optional parameter or value, angular ones represent mandatory parameter or piece of data.